molgenis/molgenis-emx2
# fix: dont crash when loading biobanks fails

### What are the main changes you did
- closes GCC#821
### How to test
- load an outdated biobank model
- go to the app
- see that instead of having an infinite spinner and console error, it now informs the user an error has occured and 0 biobanks where found.

### Checklist
